package.json

```diff
@@ -4,22 +4,25 @@
   "private": true,
   "dependencies": {
     "@capacitor/core": "1.3.0",
-    "@ionic/react": "^5.3.1",
-    "@ionic/react-router": "^5.3.1",
+    "@ionic/react": "^6.0.0-rc.2",
+    "@ionic/react-router": "^6.0.0-rc.2",
     "@types/jest": "24.0.18",
     "@types/node": "12.7.5",
     "@types/react": "^16.9.43",
     "@types/react-dom": "^16.9.8",
     "@types/react-router": "^5.1.8",
     "@types/react-router-dom": "^5.1.5",
+    "date-fns": "^2.25.0",
     "ionicons": "^5.0.1",
+    "mini-create-react-context": "^0.4.1",
     "node-sass": "^4.13.0",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
     "react-scripts": "^3.4.1",
-    "reselect": "^4.0.0"
+    "reselect": "^4.0.0",
+    "swiper": "^6.8.4"
```

Comment:
Can we use swiper 7? This is what we are going to recommend moving forward. I have some guides here: https://github.com/ionic-team/ionic-docs/blob/swiper7-support/docs/react/slides.md

Don't worry about the IonicSlides modules -- that won't work until RC 3. The big thing to note here is the correct CSS/JS imports.